src/memgraph.cpp

```diff
@@ -355,6 +356,11 @@ DEFINE_VALIDATED_string(query_modules_directory, "",
                           return true;
                         });
 
+// NOLINTNEXTLINE (cppcoreguidelines-avoid-non-const-global-variables)
+DEFINE_string(callable_mappings_path, "",
+              "The path to mappings that describe aliases to callables in cypher queries in the form of key-value "
+              "pairs in a json file.");
```

Comment:
I would explain here what the "callables" means, e.g., query module procedures and functions. In general, give a bit more context about this flag, but in additional 2 sentences max.